awx/ui_next/src/screens/Template/shared/WorkflowJobTemplateForm.jsx

```diff
@@ -0,0 +1,473 @@
+            >
+              <FieldTooltip
+                content={i18n._(
+                  t`Provide a host pattern to further constrain the list of hosts that will be managed or affected by the workflow. This limit is applied to all job template nodes that prompt for a limit. Refer to Ansible documentation for more information and examples on patterns.`
+                )}
+              />
+              <FormField type="text" name="limit" id="wfjt-limit" label="" />
+            </FormGroup>
+            <FormGroup
+              fieldId="wfjt-scmBranch"
+              id="wfjt-scmBranch"
+              label={i18n._(t`SCM Branch`)}
+              name="scmBranch"
+            >
+              <FieldTooltip
+                content={i18n._(
+                  t`Select a branch for the workflow. This branch is applied to all job template nodes that prompt for a branch.`
+                )}
+              />
+              <FormField
+                type="text"
+                id="wfjt-scmBranch"
+                label=""
+                name="scmBranch"
+              />
+            </FormGroup>
+          </FormColumnLayout>
+          <FormFullWidthLayout>
+            <Field name="labels">
+              {({ form, field }) => (
+                <FormGroup
+                  label={i18n._(t`Labels`)}
+                  name="wfjt-labels"
+                  fieldId="wfjt-labels"
+                >
+                  <FieldTooltip
+                    content={i18n._(t`Optional labels that describe this job template,
+                    such as 'dev' or 'test'. Labels can be used to group and filter
+                    job templates and completed jobs.`)}
+                  />
+                  <LabelSelect
+                    value={field.value}
+                    onChange={labels => form.setFieldValue('labels', labels)}
+                    onError={() => setContentError()}
+                  />
+                </FormGroup>
+              )}
+            </Field>
+          </FormFullWidthLayout>
+          <FormFullWidthLayout>
+            <VariablesField
+              id="wfjt-variables"
+              name="extra_vars"
+              label={i18n._(t`Variables`)}
+              tooltip={i18n._(
+                t`Pass extra command line variables to the playbook. This is the -e or --extra-vars command line parameter for ansible-playbook. Provide key/value pairs using either YAML or JSON. Refer to the Ansible Tower documentation for example syntax.`
+              )}
+            />
+          </FormFullWidthLayout>
+          <FormCheckboxLayout
+            fieldId="options"
+            isInline
+            label={i18n._(t`Options`)}
+            css="margin-top: 20px"
+          >
+            <Field
+              id="wfjt-webhooks"
+              name="hasWebhooks"
+              label={i18n._(t`Webhooks`)}
+            >
+              {({ form }) => (
+                <Checkbox
```

Comment:
I was unable to use the CheckboxField component here because I needed access to the onChange function.  This field manages whether the subform is open, and that state of the subform when the user opens and closes the form.